### PR TITLE
updates to manifest updates

### PR DESCRIPTION
**Pull request by tropicadri**
Fixes #SOME_ISSUE_NUMBER

Changes proposed in this pull request:
Icons and name can from the manifest can no longer be updated for PWAs, on mobile or desktop. 
This PR updates the corresponding article to reflect that.

### CODE REVIEW

File: src/site/content/en/progressive-web-apps/manifest-updates/index.md [hunk 1]
```diff
@@ -104,18 +100,12 @@ used.
 
 * `background_color`
 * `display`
-* `icons`
-* `name`
 * `orientation`
 * `scope`
 * `shortcuts`
-* `short_name`
 * `start_url`
```
Comment (chasephillips) [inline src/site/content/en/progressive-web-apps/manifest-updates/index.md:106]: `start_url` is mentioned above as being excluded from update so probably should be removed here, too.

File: src/site/content/en/progressive-web-apps/manifest-updates/index.md [hunk 2]
```diff
@@ -41,19 +41,15 @@ compare it against the local copy.
...
 * `display` (see below)
-* `icons`
 * `scope`
 * `shortcuts`
 * `theme_color`
-* Pixel changes to primary icon file
-* Pixel changes to splash icon file
 
 {% Aside 'caution' %}
 Changes to `name`, `short_name`, and `start_url` are **not** currently
```
Comment (chasephillips) [inline src/site/content/en/progressive-web-apps/manifest-updates/index.md:55]: We might want to mention `icons` and related icon fields here.

**Comment by jpmedley** [comment]
@tropicadri Let me know when all your technical reviews are done. I'll do the typo and writing review then

**Comment by tropicadri** [comment]
@jpmedley will do! Still working on details with eng and product.

File: src/site/content/en/progressive-web-apps/manifest-updates/index.md (hunk 1 above)
Comment (tropicadri) [reply src/site/content/en/progressive-web-apps/manifest-updates/index.md:107]: it is excluded on desktop, but still supported on mobile, some changes are coming but will update the article when those happen.

File: src/site/content/en/progressive-web-apps/manifest-updates/index.md (hunk 2 above)
Comment (tropicadri) [reply src/site/content/en/progressive-web-apps/manifest-updates/index.md]: done, thank you!

**Comment by tropicadri** [comment]
@jpmedley ready now! Thank you :)

**Review by jpmedley — APPROVED** [review]

**Pull request merged**